packages/dashboard-frontend/src/store/Plugins/devWorkspacePlugins/index.ts

```diff
@@ -94,56 +125,83 @@ export const actionCreators: ActionCreators = {
     }
   },
 
-  requestDwDefaultEditor: (settings: che.WorkspaceSettings): AppThunk<KnownAction, Promise<void>> => async (dispatch): Promise<void> => {
-    const pluginRegistryUrl = settings.cheWorkspacePluginRegistryUrl;
-    const defaultEditor = settings['che.factory.default_editor'];
+  requestDwEditor: (settings: che.WorkspaceSettings, editorName: string): AppThunk<KnownAction, Promise<void>> => async (dispatch): Promise<void> => {
+    dispatch({
+      type: 'REQUEST_DW_EDITOR',
+      editorName,
+    });
 
-    if (!pluginRegistryUrl || !defaultEditor) {
-      const errorMessage = 'Failed to load the default editor, reason: plugin registry URL or default editor ID is not provided by Che server.';
-      dispatch({
-        type: 'RECEIVE_DW_DEFAULT_EDITOR_ERROR',
-        error: errorMessage,
-      });
-      throw errorMessage;
-    }
+    let editorUrl;
+    // check if the editor is an id or a path to a given editor
+    if (editorName.startsWith('https')) {
```

Comment:
```suggestion
    if (editorName.startsWith('https://')) {
```
could be a bit safer )

Reply:
agree, fixed